ajax/getUsers.php

```diff
@@ -1,11 +1,13 @@
 <?php
-OCP\User::checkLoggedIn ();
-OCP\JSON::callCheck ();
+
+OCP\User::checkLoggedIn();
+OCP\JSON::callCheck();
+
+header('Content-Type: application/json; charset=utf-8');
 
 $limit = 10;
 $offset = 0;
 
-$users = OCP\User::getDisplayNames((string)$_GET['search'], $limit, $offset);
+$users = OCP\User::getDisplayNames((string) $_GET['search'], $limit, $offset);
 
-OCP\JSON::encodedPrint ( $users );
-?>
+json_encode($users);
```

Comment:
I don't think this will work. You'll have to echo the output and set the correct header. https://github.com/owncloud/core/blob/master/lib/private/json.php#L172 Or do I miss something?